hack/update-codegen.sh

```diff
@@ -37,7 +37,9 @@ ${GOPATH}/bin/deepcopy-gen --input-dirs \
   --go-header-file ${SERVING_ROOT}/hack/boilerplate/boilerplate.go.txt
 
 # Update code to change Gatewaies -> Gateways to workaround cleverness of codegen pluralizer.
-find -name '*.go' -exec grep -l atewaies {} \; | xargs sed 's/atewaies/ateways/g' -i
+find . -name '*.go' -exec grep -l atewaies {} \; | xargs sed -i -e 's/atewaies/ateways/g'
```

Comment:
Interesting, I thought you would need `-i ''` on macos, like in https://github.com/knative/serving/pull/1694/files#diff-5345a009f642e55fc0d5b16d4f5d6f9bR40